### ISSUE TITLE
GridFeaturizer Fingerprints Broken?

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#334]
I've been working on fixing up the `GridFeaturizer` example and integrating it into PDBBind. As of right now though, it looks like the fingerprints might be broken. As @prasadkawthekar reported offline, I'm getting R^2 near 1. on the training set, but R^2 close to 0. on the validation set. I've tested this so far on the refined set (~3500 complexes) and am in the process of testing on the full set. @evanfeinberg's old code had much better numbers, so it's possible that something broken between now and then. It might possibly be due to `openbabel` issues (#329, #331, #333).

@evanfeinberg: Do you have any suggestions for extra unit tests we can add to check that `GridFeaturizer` is functional?

CC @joegomes @miaecle: I'd recommend not working with the pdbbind `GridFeaturizer` code until these issues are resolved.

**Comment by rbharath** [comment]
Here are numbers on the full set with index-based splitting

```
Train scores
{'pearson_r2_score': 0.96576559535181217}
Validation scores
{'pearson_r2_score': 0.004389175528761744}
```

These numbers are very strange. The fact that the training score is high indicates that there's signal in the featurization (and when I checked, the featurization seems to output reasonable looking vectors), but it's strange the validation scores are so low then.

**Comment by rbharath** [comment]
Closing this issue. I think it's an issue with the split and not with the fingerprints themselves. @joegomes, @miaecle Feel free to work with the datasets using the new amenities in #335.

**Comment by evanfeinberg** [comment]
That's very reassuring, great find @rbharath. I'll take another look at
this in a week or so.
